src/styles/_fonts.scss

```diff
@@ -1,6 +1,8 @@
-@font-face {
-  font-family: "Roboto";
-  src: url("../fonts/Roboto-Regular-webfont.woff") format("woff");
-  font-weight: normal;
+.page {
+  font-family: Roboto, sans-serif;
   font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 27px;
+  color: #3e3e3e;
```

Comment:
you are describing the `page` block and it should be in `page.scss` file. `font.scss` just imports fonts